.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Automatically request reviews
+dask/dataframe/dask_pandas.py    @kkraus14 @jakirkham @rjzamora
```

Comment:
I'd prefer to make this a team instead of specific individuals. It will allow us to more easily add/remove folks

Reply:
Yeah that works for me. I am torn between using dask/gpu,  dask/dataframe, or making a new one that is the intersection.

Reply:
Also, from [GitHub's docs on code owners](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners), it looks like code owners (whether they are individuals or teams) must have write permissions to the repository. In order to keep notifications and permissions unrelated, I'll propose we create a `dataframe-dispatch` team (or some other name) that we can manually ping, but hold off on creating a `.github/CODEOWNERS ` file for the time being. This is less ideal than an automated approach, but I suspect will still work in practice

Reply:
Yep agree with this being a team as well

Reply:
https://github.com/dask/dask/pull/7506